src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -58,6 +58,45 @@
  * <pre>
  * &lt;module name=&quot;SingleLineJavadoc&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *   /** &#64;return integer value *&#47;
+ *   public int sum(int a, int b) {
+ *     return a + b;
+ *   }
+ * } //Violation. If Javadoc contains at least one at-clause, it should be formatted in multiple lines. 
```

Comment:
please place `// violation` comment on line were violation is placed by CLI tool.
it should be somethig like:
`/** &#64;return violation, when at least one at-clause, it should be formatted in multiple lines. *&#47;`